src/main/java/com/practise/tdd/PerfectNumberValidator.java

```diff
@@ -1,10 +1,48 @@
 package com.practise.tdd;
 
+import java.util.ArrayList;
+import java.util.List;
 
 public class PerfectNumberValidator {
 
 	public boolean isPerfectNumber(int number) {
+		List<Integer> divisorsOfNumber = getAllDivisors(number);
+		
+		if(number!= 1 && sumOfDivisorsEqualsNumber(divisorsOfNumber, number))
+			return true;
+		
```

Comment:
Why 1? sumOfDivisorsEqualsNumber() return type is boolean. 
Could have been 
return number!= 1 && sumOfDivisorsEqualsNumber(divisorsOfNumber, number);